core/src/main/scala/com/yahoo/maha/core/BaseUTCTimeProvider.scala

```diff
@@ -52,15 +52,17 @@ class BaseUTCTimeProvider extends UTCTimeProvider with Logging {
     var utcDayFilter = localTimeDayFilter
     var utcHourFilter = localTimeHourFilter
     var utcMinuteFilter = localTimeMinuteFilter
-    if (isDebugEnabled) info(s"Timezone: $timezone")
+    info(s"Timezone: $timezone")
     if (!validateFilters(localTimeDayFilter, localTimeHourFilter, localTimeMinuteFilter) || !timezone.isDefined) {
+      info(s"validateFilters is "+ validateFilters(localTimeDayFilter, localTimeHourFilter, localTimeMinuteFilter) + "timezone is defined " + timezone.isDefined)
```

Comment:
store previous result of validateFilters before calling it twice? and fix string interpolations

Reply:
fixed